.github/workflows/ci.yml

```diff
@@ -31,7 +31,6 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2.4.0
       - name: Azure login
-        if: matrix.os-name == 'Windows'
```

Comment:
@danielmarbach why do we need a storage account when running on Linux?

Reply:
The question should be: why do we need to log in on Linux. The answer is: we don't and the heat today made me confused. I'll bring back the line and then we can release a patch together with your fixes

Reply:
Here we go https://github.com/Particular/setup-oracle-action/pull/7

Reply:
Ah, I made the comment in the context of [this commit](https://github.com/Particular/setup-oracle-action/pull/3/commits/f0c7775211602823479324782b4f0592d1b2981b) but I guess you're looking at a different view.

BTW ­— this doesn't need to be in a patch, because it's just the CI build. It doesn't affect the actual action. FYI I already tagged an earlier commit with v1.0.1, which contains my logging fix.

Reply:
>  this doesn't need to be in a patch, because it's just the CI build

I'm aware of that. I wasn't aware though that you have already tagged master with the other changes. That's why I was mentioning tagging (potentially including this change too)